backend/utils.py

```
@@ -0,0 +1,113 @@
+        "password123", "admin123", "letmein", "welcome1"
+    ]
+    
+    if password.lower() in common_passwords:
+        errors.append("Password is too common and easily guessable")
+    
+    if errors:
+        return False, errors
+    
+    return True, []
+
+def calculate_password_strength(password):
```

Pylint (3.3.7)

[refactor] Too many branches (14/12)

(R0912)